### PR TITLE
[JOSS] Paper suggestions

### PR DESCRIPTION
**Pull request by RMeli**
https://github.com/openjournals/joss-reviews/issues/6607

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -43,29 +43,29 @@ Here are a few examples:
 - [`GPUE`](https://github.com/GPUE-group/GPUE) [@Schloss2018] for 1D to 3D Gross-Pitaevskii equations accelerated on GPU, in C++ (currently unmaintained).
 - [`py-fmas`](https://github.com/omelchert/py-fmas) for 1D NLSE in optical fibers, with a split-step method (currently unmaintained).
 
-With our project, we bring similar performance to C++ and Julia implementations, while striving for accessibility and maintainability by using the prevalant language in the physics community, Python.
```
Comment (RMeli) [inline paper/paper.md:46]: This is a big claim, without any evidence. I'd simply remove it.

File: paper/paper.md [hunk 2]
```diff
@@ -43,29 +43,29 @@ Here are a few examples:
...
+With our project, we bring similar performance to C++ and Julia implementations, while striving for accessibility and maintainability by using Python.
 Using an easy to extend object-oriented classes, users can readily input experimental parameters to quickly model real setups.
 
 # Functionality
 
-`NLSE` harnesses the power of pseudo-spectral schemes in order to solve efficiently the following general type of equation:
```
Comment (RMeli) [inline paper/paper.md:51]: I changed all "in order to" to "to". I use "in order to" a lot (I'm an Italian native speaker), but I've been told that it's too verbose for no reason and using "to" is better.

File: paper/paper.md [hunk 3]
```diff
@@ -43,29 +43,29 @@ Here are a few examples:
...
 It supports lossy, non-linear and non-local media.
 
-It provides several classes to model 1D, 2D or 3D propagation, and leverages the array functionalities of `Numpy` like broadcasting in order to allow scans of physical parameters to most faithfully replicate experimental setups.
+It provides several classes to model 1D, 2D or 3D propagation, and leverages the array functionalities of `Numpy` like broadcasting to allow scans of physical parameters to most faithfully replicate experimental setups.
 The typical output of a simulation run is presented in \autoref{fig:output}.
 
-This code has been developped during the author's PhD thesis [@aladjidiFullOpticalControl2023] and used as the main simulation tool for several publications like [@glorieuxHotAtomicVapors2023] and [@bakerrasooliTurbulentDynamicsTwodimensional2023].
+This code has been developed initially developed in @aladjidiFullOpticalControl2023 and used as the main simulation tool for several publications like @glorieuxHotAtomicVapors2023 and @bakerrasooliTurbulentDynamicsTwodimensional2023.
```
Comment (RMeli) [inline paper/paper.md:66]:
```suggestion
This code has been developed initially in @aladjidiFullOpticalControl2023 and used as the main simulation tool for several publications like @glorieuxHotAtomicVapors2023 and @bakerrasooliTurbulentDynamicsTwodimensional2023.
```
Whoops.

**Comment by taladjidi** [comment]
Thank you for your comments 👍 I just merged them. I'll try to fix the DOI now

**Pull request merged**